docs/resources/oracledb_session.md.erb

```diff
@@ -20,11 +20,17 @@ A `oracledb_session` resource block declares the username and password to use fo
 where
 
 * `oracledb_session` declares a username and password with permission to run the query (required), and an optional parameters for host (default: `localhost`), SID (default: `nil`, which uses the default SID, and path to the sqlplus binary (default: `sqlplus`).
+* it is possible to run queries as sydba/sysoper by set password to value 'as sysdba', see examples
 * `query('QUERY')` contains the query to be run
 * `its('value') { should eq('') }` compares the results of the query against the expected result in the test
 
 <br>
 
+## Resource Properties
+* rows the query result as array of hashes
+* row(id)  selected row from query result
```

Comment:
Also note that `rows` returns a simple Array of Hashes, while `row[index]` returns a SQLRow object, which is a much wierder thing than a hash.  It seems odd that they are different.

If you want to ease column / cell access, you might implement `[]` on the SQLRows and SQLColumn Classes.

Reply:
I'll answer here on two comments about rows method. I have not added any examples for rows because my usage of it is not really clean inspec. I'm using it in ruby code to build another checks based on SQL query result, i.e. when I need to use data out of (remote) DB for my checks.
For example something like this:
```ruby
sql.query('SELECT * FROM files;').rows.each do |file_row|
  describe file(file_row['path']) do
    its('owner') { should eq file_row['owner']}
  end
end
```
If you think that such unconventional example may be added to documentation I'll do it. 

This is also answer to why rows return plain array of hashes - in construction above hashes are easier to use and offers more possibilities, that array of SQLRows.

Reply:
I think that's a perfectly valid example.  You might offer it as a second, more advanced example.  As a first example, you might do:
```
describe sql.query('SELECT * FROM files;').rows
  its('count') { should eq 20 }
end
```

Reply:
To check the number of rows in result you can do:
```ruby
describe sql.query('SELECT * FROM files;') do
  its('size') { should eq 20 }
end
```
But I surely would agree, that using rows and count make it much more readable.